### PR TITLE
feat(carousel): allow to display multiple items per one slide

### LINKED ISSUE #965: feat(carousel): add support of multiple items in slide
Are you planing to allow multiple items in one slide ?
Like this one: https://codepen.io/anon/pen/YGqKzR

### PR DESCRIPTION
**Pull request by musienkoyuriy**
Closes #965
# PR Checklist
Before creating new PR, please take a look at checklist below to make sure that you've done everything that needs to be done before we can merge it.

 - [ ] read and followed the [CONTRIBUTING.md](https://github.com/valor-software/ngx-bootstrap/blob/development/CONTRIBUTING.md) guide.
 - [ ] built and tested the changes locally.
 - [ ] added/updated tests.
 - [ ] added/updated API documentation.
 - [ ] added/updated demos.

### CODE REVIEW

**Comment by ludmilanesvitiy** [comment]
1. Open Carousel demo page, take a look at console log:
Screenshot from 2019-04-15 10-28-19

2. Broken existed functionality, STR:
- Open https://valor-software.com/ngx-bootstrap/#/carousel#captions
-  When shown the First slide, click on left arrow "<"
-  Observe the results: latest (Third) slide is active

In current implementation: first slide item will not switch to third.
Peek 2019-04-15 10-41

3. Similar issue with switching item from 3d to the 1st. (reproduced by tests in Disable slide looping)

File: package.json [hunk 1]
```diff
@@ -50,7 +50,8 @@
     "serve:ssr": "node demo/dist/server",
     "build:client-and-server-bundles": "ng build --prod && ng run ngx-bootstrap:server",
     "webpack:server": "webpack --config ./scripts/universal/webpack.server.config.js --progress --colors",
-    "configure-heroku": "node scripts/universal/configure-heroku.js"
+    "configure-heroku": "node scripts/universal/configure-heroku.js",
+    "build.carousel": "rimraf dist/carousel && node scripts/ng-packagr/api ../../src/carousel/package.json"
```
Comment (ludmilanesvitiy) [inline package.json]: why do we need it?

**Comment by ludmilanesvitiy** [comment]
Rechecked existed and new functionality - looks good. Smoke tested in Chrome, FF, IE.

File: package.json (hunk 1 above)
Comment (musienkoyuriy) [reply package.json]: it was just for my personal convenience and it is temporary (to build just my module during the work with it). As all is ok, I going to delete it

File: src/carousel/carousel.component.ts [hunk 2]
```diff
@@ -95,6 +118,18 @@ export class CarouselComponent implements OnDestroy {
     Object.assign(this, config);
   }
 
+  ngAfterViewInit(): void {
+    setTimeout(() => {
```
Comment (Domainv) [inline src/carousel/carousel.component.ts:120]: Are you sure we can't do without it?

**Review by Domainv — CHANGES_REQUESTED** [review]

File: src/carousel/carousel.component.ts (hunk 2 above)
Comment (Domainv) [reply src/carousel/carousel.component.ts]: check

**Review by Domainv — DISMISSED** [review]

**Comment by ludmilanesvitiy** [comment]
Smoke retested in Chrome, Edge - looks good.

**Review by Domainv — APPROVED** [review]

**Pull request merged**